src/Aquifer.Public.API/Endpoints/Bibles/Alignments/Greek/Endpoint.cs

```diff
@@ -15,10 +15,13 @@ public override void Configure()
     {
         Get("/bibles/{BibleId}/alignments/greek");
         Options(EndpointHelpers.UnauthenticatedServerCacheInSeconds(EndpointHelpers.OneHourInSeconds));
-        Description(d => d.ProducesProblemFE());
+        Description(d => d
+            .WithTags("Bibles")
```

Comment:
We _could_ move this into a constant in a shared location but being that the docs will break if it is incorrect and the routes are not using constants this seemed fine.